src/redist/targets/packaging/windows/clisdk/bundle.wxs

```diff
@@ -14,23 +14,6 @@
         WixBundleInstalled OR ((NOT (DOTNETHOME_X64 ~= DOTNETHOME_X86)) OR DOTNETHOMESIMILARITYCHECKOVERRIDE)
     </bal:Condition>
 
-    <!-- ***** When the product-band version: "3.0.1xx" advances, remove this section ***** -->
```

Comment:
Is Preview 4 not in `3.0.1xx`? Should the PR title really be "Remove the hard-coded Preview 3 upgrade codes for `3.0.2xx`"?

Want to make sure, because I might be misunderstanding `RelatedBundle` otherwise.